client/web/compose/src/components/PageBlocks/TabsConfigurator.vue

```diff
@@ -109,137 +109,137 @@
       </h5>
     </div>
 
-    <b-table-simple
-      v-if="block.options.tabs.length"
-      responsive="lg"
-      borderless
-      small
+    <c-form-table-wrapper
+      :labels="{
+        addButton: $t('general:label.add')
+      }"
+      @add-item="addTab"
     >
-      <b-thead>
-        <tr>
-          <th />
-
-          <th class="d-flex align-items-center text-primary">
-            {{ $t('tabs.table.columns.title.label') }}
-            <c-hint
-              :tooltip="$t('interpolationFootnote', ['${record.values.fieldName}', '${recordID}', '${ownerID}', '${userID}'])"
-              class="d-block"
-            />
-          </th>
-
-          <th
-            class="text-primary"
-          >
-            {{ $t('tabs.table.columns.block.label') }}
-          </th>
-
-          <th />
-        </tr>
-      </b-thead>
-
-      <draggable
-        v-model="block.options.tabs"
-        handle=".handle"
-        tag="b-tbody"
+      <b-table-simple
+        v-if="block.options.tabs.length"
```

Comment:
This v-if makes is look very strange, did you test this?
Remove the v-if since table should be there. You should also remove the no tabs section